src/Promenade.Web/Views/Home/Locations.cshtml

```diff
@@ -0,0 +1,160 @@
+                                                    <tr>
+                                                        <td style="padding-bottom:10px;">
+                                                            <strong style="font-size: 17.5px;">
+                                                                @Model.CloseLocations[i].Name
+                                                            </strong> <em style="font-size:12px;">@if (Model.CloseLocations[i].Distance != 0){<text>(@Model.CloseLocations[i].Distance mi.)</text>}</em>
+                                                        </td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td style="padding-bottom:10px;font-size:13px;">
+                                                            @Model.CloseLocations[i].Address<br>
+                                                            @Model.CloseLocations[i].City  AZ, @Model.CloseLocations[i].Zip
+                                                        </td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td style="padding-bottom:10px;">
+                                                            <strong class="traffic-green">Open until 5:00 PM</strong>
+                                                        </td>
+                                                    </tr>
+                                                    <tr style="text-align: left;">
+                                                        <td style="padding: 5px 0;">
+                                                            <a href="Locations/@Model.CloseLocations[i].Stub" target="_blank" class="btn btn-default">Library Info</a>
+                                                            <a href="@Model.CloseLocations[i].EventLinkId" class="btn btn-default" style="float:right" target="_blank">Events</a>
+                                                        </td>
+
+                                                    </tr>
+                                                </tbody>
+                                            </table>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </text>
+                    }
+                }
+                else
+                {
+                    <script>
+                        var noResults = true;
+                    </script>
+                }
+            </div>
+        </div>
+        @section scripts {
+            <script>
+        var useMyLocation = 'Use my location';
+
+        $().ready(function () {
+            if (location.protocol != 'https:' && location.hostname != 'localhost') {
```

Comment:
Do we need to do this? It is ideal to not have to put a bunch of display logic surrounding whether TLS is being used.

Reply:
This is the same logic that was used int the Geolocator. Unsure of why its needed but it breaks without it